cpp/log/etcd_consistent_store-inl.h

```diff
@@ -648,32 +626,38 @@ util::Status EtcdConsistentStore<Logged>::CleanupOldEntries() {
   LOG(INFO) << "Cleaning old entries up to and including sequence number: "
             << clean_up_to_sequence_number;
 
-  std::vector<EntryHandle<Logged>> sequenced_entries;
-  util::Status status(GetSequencedEntries(&sequenced_entries));
+  EntryHandle<ct::SequenceMapping> sequence_mapping;
+  util::Status status(GetSequenceMapping(&sequence_mapping));
   if (!status.ok()) {
-    LOG(WARNING) << "Couldn't get sequenced entries: " << status;
+    LOG(WARNING) << "Couldn't get sequence mapping: " << status;
     return status;
   }
 
-  for (auto& entry : sequenced_entries) {
-    const uint64_t sequence_number(entry.Entry().sequence_number());
-
-    // return if we've hit the limit
-    if (entry.Entry().sequence_number() > clean_up_to_sequence_number) {
-      return util::Status::OK;
-    }
+  size_t i;
```

Comment:
Or just, you know, use this one we've already got without the fear of being off-by-one? :)
On the plus side, I've dinged all the swapping stuff, and added a few more comments which has made it a bit clearer.

`size_t` is unsigned, but I suspect you meant *use signed* :)